man/ocitools-generate.1.md

```diff
@@ -153,26 +153,43 @@ inside of the container.
 **--rootfs**=ROOTFSPATH
   Path to the rootfs
 
-**--seccomp-arch**=ARCH
-  Specifies Additional architectures permitted to be used for system calls.
-  By default if you turn on seccomp, only the host architecture will be allowed.
-
-**--seccomp-default**=ACTION
-  Specifies the the default action of Seccomp syscall restrictions
-  Values: KILL,ERRNO,TRACE,ALLOW
-
-**--seccomp-syscalls**=SYSCALLS
-  Specifies Additional syscalls permitted to be used for system calls,
-  e.g Name:Action:Arg1_index/Arg1_value/Arg1_valuetwo/Arg1_op, Arg2_index/Arg2_value/Arg2_valuetwo/Arg2_op
-  See --seccomp-allow and --seccomp-errno for convenient way to set seccomp syscall options.
-
-**--seccomp-allow**=SYSCALL
-  Specifies syscalls to be added to the ALLOW list.
-  See --seccomp-syscalls for setting limits on arguments.
-
-**--seccomp-errno**=SYSCALL
-  Specifies syscalls to be added to the ERRNO list.
-  See --seccomp-syscalls for setting limits on arguments.
+  **--seccomp-arch**=ARCH
+    Specifies Additional architectures permitted to be used for system calls.
```

Comment:
You're increasing the indent (the rest of these entries have no whitespace before the initial `**`, and only two spaces before subsequent lines.